WaterNetworkManagement/Simulation/Simulation/doc/spec.md

```diff
@@ -12,36 +12,31 @@ A JSON Schema corresponding to this data model can be found [here](../schema.jso
 
 -   `type`: Entity type. It must be equal to `Simulation`.
 
--   `modifiedAt`: Last update timestamp of this
-    entity.
-
-    -   Attribute type: Property. [DateTime](https://schema.org/DateTime)
-    -   Read-Only. Automatically generated.
-
--   `createdAt`: Entity's creation timestamp.
-
-    -   Attribute type: Property. [DateTime](https://schema.org/DateTime)
-    -   Read-Only. Automatically generated.
-
 ### Simulation Entity Properties
 
 -   `description` : A free text description
     -   Attribute type: `Property`.Text
     -   Optional
 
--   `optionNames` : Names of options applied in the simulation
```

Comment:
I switched it to NGSI-LD properties for making queries such as "get the simulation accuracy" faisable.  I will add these properties to the schema, then maybe it will be good if you can add examples in the payload;

Reply:
`qualityTimeStep` still missing from schema and spec